src/core/plugins/core-fingerpoint/client/index.ts

```diff
@@ -0,0 +1,195 @@
+import * as alt from 'alt-client';
+import * as natives from 'natives';
+
+let FINGERPOINT_KEY = 66
+
+export class Fingerpointing {
```

Comment:
Doesn't actually need to be a class at all. Can just be a const object.

See: #299